zrml/neo-swaps/src/lib.rs

```diff
@@ -740,6 +801,48 @@ mod pallet {
             )
         }
 
+        /// Deploy a combinatorial pool for the specified markets and provide liquidity.
+        ///
+        /// The tokens of each of the markets specified by `market_ids` are split into atoms. For
+        /// each combination of outcome tokens `x, ..., z` from the markets, there is one
+        /// combinatorial token `x & ... & z` in the pool.
+        ///
+        /// The pool's assets are ordered by lexicographical order, using the ordering of tokens of
+        /// each individual market provided by the `MarketCommonsApi`. For example, if three markets
+        /// with outcomes `x_1, x_2`, `y_1, y_2` and `z_1, z_2` are involved, the outcomes of the
+        /// pool are (in order):
+        ///
+        /// x_1 & y_1 & z_1
+        /// x_1 & y_1 & z_2
+        /// x_1 & y_2 & z_1
+        /// x_1 & y_2 & z_2
+        /// x_2 & y_1 & z_1
+        /// x_2 & y_1 & z_2
+        /// x_2 & y_2 & z_1
+        /// x_2 & y_2 & z_2
+        ///
+        /// The sender specifies a vector of `spot_prices` for the assets of the new pool, in the
+        /// order as described above.
+        ///
+        /// Depending on the values in the `spot_prices`, the transaction will transfer different
+        /// amounts of each outcome to the pool. The sender specifies a maximum `amount` of outcome
+        /// tokens to spend.
+        ///
+        /// Unlike in the `deploy_pool` extrinsic, the sender need not acquire the outcome tokens
+        /// themselves. Instead, all they need is `amount` units of collateral.
+        ///
+        /// Deploying the pool will cost the signer an additional fee to the tune of the
+        /// collateral's existential deposit. This fee is placed in the pool account and ensures
+        /// that swap fees can be stored in the pool account without triggering dusting or failed
+        /// transfers.
+        ///
+        /// The `force_max_work` parameter can be used to force the `CombinatorialTokensApi` to
+        /// spend the maximum amount of work, independently of the parameters that it is called
+        /// with. This is useful for benchmarking purposes and should not be used in production.
```

Comment:
It will be called `Fuel` after the audit fixes.

Reply:
Fixed here https://github.com/zeitgeistpm/zeitgeist/pull/1391/commits/013dc55bc5dc5cadb56df7ed53c0a77658b0f031